### PR TITLE
Update README.md for .NET Core 3

### PR DESCRIPTION
**Pull request by hartmair**
Add Moving to .NET Core section regarding to https://github.com/dotnet/vblang/issues/448#issuecomment-504223057

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,7 +10,15 @@ If you discover bugs or deficiencies in the above, please leave an issue to rais
 
 For *new feature proposals*, however, please raise them for [discussion](https://github.com/dotnet/vblang/labels/Discussion), and *only* submit a proposal as a pull request if invited to do so by a member of the Language Design Team (a "champion").
 
-## Discussion
+## Contribute
```
Comment (KathleenDollard) [inline README.md]: I'd like to drop Contribute here and raise "Moving to .NET Core" to a two hashmark level

**Comment by AdamSpeight2008** [comment]
Could you also add links to the Gitter discussion pages, as it initially looks like VB.Net doesn't have one when it does.

**Comment by KathleenDollard** [comment]
@AdamSpeight2008 Can you create an issue for this? 

Including links to other sites is a great idea. 

I'd like some discussion on various community sites and opportunities because I'd prefer not to have this site seem opinionated on that.

**Comment by AdamSpeight2008** [comment]
@KathleenDollard @hartmair  already has #457

**Comment by Code-DJ** [comment]
@KathleenDollard not sure if this is off-topic for readme but given that VSCode, VS for Mac don't support VB.NET yet, a word about if and when it would be supported would be nice.

**Pull request closed without merging**